docs/developer-guide/firstanalysis.md

```diff
@@ -79,4 +79,4 @@ For example, have a look at the following program: `tests/regression/99-tutorial
 _Hint:_
 The easiest way to do this is to use the powerset lattice of `{-, 0, +}`.
 For example, "non-negative" is represented by `{0, +}`, while negative is represented by `{-}`.
-To do this, modify `SL` by using `SetDomain.Make` instead of `Lattice.Flat` and reimplementing the two functions using `singleton` and `for_all`.
+To do this, modify `SL` by using `SetDomain.FiniteSet` (takes a `struct` with a list of finite elements as second parameter) instead of `Lattice.Flat` and reimplementing the two functions using `singleton` and `for_all`.
```

Comment:
I don't recall having an issue with `SL.top` when solving the signs tutorial. Is it a problem on the existing signs tutorial tests or some other programs? If the latter, then maybe it should be covered by a test.

Reply:
I think it was one of the existing programs. Anyway, I think that this solution is the nicer one anyway.